lib/importproject.cpp

```diff
@@ -621,8 +621,16 @@ void ImportProject::importVcxproj(const std::string &filename, std::map<std::str
                 for (const tinyxml2::XMLElement *cfg = node->FirstChildElement(); cfg; cfg = cfg->NextSiblingElement()) {
                     if (std::strcmp(cfg->Name(), "ProjectConfiguration") == 0) {
                         const ProjectConfiguration p(cfg);
-                        if (p.platform != ProjectConfiguration::Unknown)
+                        if (p.platform != ProjectConfiguration::Unknown) {
                             projectConfigurationList.emplace_back(cfg);
+                            bool alreadyIn = false;
```

Comment:
hmm.. I envisioned that this would be kept out of `importVcxproj`.

    std::set<std::string> ImportProject::getVSConfigs() const {
        std::set<std::string> ret;
        for (const FileSettings &fs: fileSettings) {
            ret.insert(fs.cfg);
        }
        return ret;
    }

I guess I don't have a strong opinion, I could allow that we have it in `importVcxproj`... but at least use `std::set` instead so you don't need the extra code to avoid duplicates.